src/components/appSwitch/AppSwitch.js

```diff
@@ -37,7 +36,11 @@ class AppSwitch extends React.Component<LoadingProps, {}> {
    */
   constructor(props: LoadingProps) {
     super(props)
-    this.checkAuthStatus()
+    this.ready = false
+    this.checkAuthStatus().then(r => {
+      this.ready = true
+      this.forceUpdate()
+    })
```

Comment:
```suggestion
    })
    .catch(() => (this.ready = true))
```

or probably changing the whole thing to:

```javascript
this.checkAuthStatus()
  .then(() => this.forceUpdate())
  .catch(e => log.error('constructor', { e }))
  .then(() => this.ready = true)
```

to be sure that it always will change the flag.

Reply:
not sure i want it to stop if there's an error